client/scss/components/_button.scss

```
@@ -151,6 +151,62 @@
 
     &.button-secondary {
       border-color: theme('colors.black-20');
+      color: $color-button;
+      background-color: transparent;
+      border: 1px solid $color-button;
+
+      .icon-wrapper {
+        border-right: 1px solid $color-button;
+        background-color: transparent;
+      }
+
+      &:hover {
+        background-color: theme('colors.secondary.50');
+        color: $color-button;
+      }
+
+      &:disabled,
+      &[disabled] {
+        background-color: transparent;
+        border-color: $color-grey-3;
+        color: $color-grey-3;
+        cursor: default;
+        pointer-events: none;
+
+        .icon-wrapper {
+          border-right: 1px solid $color-grey-3;
+          background-color: transparent;
+        }
+      }
+
+      &.button-small {
+        border: none;
+        padding: 0 30px 30px;
+
+        &:hover {
+          background-color: transparent;
+          color: $color-button;
+        }
+
+        .icon-wrapper {
+          border: 2px solid $color-button;
+          border-radius: 50%;
+          border-color: $color-button;
+          background-color: transparent;
+          height: fit-content;
+          width: fit-content;
+
+          &:hover {
+            background-color: $color-button;
+            color: $color-white;
+          }
+        }
+
+        &:disabled,
+        &[disabled] {
+          display: none;
+        }
```

I don't believe this was the intention when @lb- said the disabled state was "not shown". I don't think we want to simply hide secondary icon buttons that are disabled, but there is not a design for them in the Wagtail Design System. It we show them, it looks like the only thing we need to do for it to look decent is to fix the border color:

```suggestion
        &:disabled .icon-wrapper,
        &[disabled] .icon-wrapper,
        &.disabled .icon-wrapper {
          border-color: $color-grey-4;
        }
```